.github/workflows/terraform-ci.yaml

```diff
@@ -83,62 +81,23 @@ jobs:
           role-to-assume: ${{ secrets[format('GHA_3WARE_OIDC_{0}', env.ENVIRONMENT)] }}
           role-session-name: ${{ github.event_name == 'merge_group' && 'aws-net-sec-terraform-apply' || 'aws-net-sec-terraform-plan' }}
 
+      - name: Terraform Fmt + Init + Validate
+        if: ${{ github.event_name == 'pull_request' }}
+        id: tf
+        uses: devsectop/tf-via-pr@9dfb2360a5b7d5c3385276221e6e372467199e46 # v12.0.3
+        with:
+          working-directory: ${{ matrix.targets }}
+          arg-lock: false
+          format: true
+          validate: true
+
```

Comment:
I tried using your action for format and validate, but it failed - this was in two different jobs though, so it may not occur in a single job. The error I saw, and a link to the run log is here: https://github.com/3ware/aws-network-speciality/pull/53

Reply:
There's a fair bit of difference between the workflows, so I reckon best bet is try out the latest iteration here, following your approval.